scripts/zones/Batallia_Downs/mobs/Skirling_Liger.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- Area: Batallia Downs (105)
+--  MOB: Skirling Liger
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+end;
+
+-----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    if (mob:getMod(MOD_REGAIN) ~= 0) then
+        mob:setMod(MOD_REGAIN,0);
+    end
+end;
+
+-----------------------------------
+-- onMobFight
+-----------------------------------
+
+function onMobFight(mob, target)
+    mob:setMod(MOD_REGAIN, 50);
```

Comment:
on repop mobs get reset to their initial values (whatever is in database+onMobinit+onMobSpawn) or should, if they don't then an old bug recurred and would need reported again.